From c9937faf1e2c81005289e18974ed5d10b2dc37ab Mon Sep 17 00:00:00 2001
From: Carlos Garnacho <carlosg@gnome.org>
Date: Sat, 12 Aug 2017 11:52:28 +0200
Subject: [PATCH] clutter: Fix build with --enable-debug

Message was poking stage_x11, which doesn't exist in this context.
Just print the Window that is receiving the event, the event will be
emitted into the only existing stage anyway.
---
 clutter/clutter/x11/clutter-device-manager-xi2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/clutter/x11/clutter-device-manager-xi2.c b/clutter/clutter/x11/clutter-device-manager-xi2.c
index 1ecec02..da3457a 100644
--- a/clutter/clutter/x11/clutter-device-manager-xi2.c
+++ b/clutter/clutter/x11/clutter-device-manager-xi2.c
@@ -1160,7 +1160,7 @@ translate_pad_event (ClutterEvent       *event,
                 event->any.type == CLUTTER_PAD_RING
                 ? "pad ring  "
                 : "pad strip",
-                (unsigned int) stage_x11->xwin,
+                (unsigned int) xev->event,
                 device->id,
                 device->device_name,
                 event->any.time, value);
--
libgit2 0.26.0

